core/primitives/src/congestion_info.rs

```diff
@@ -532,203 +444,231 @@ mod tests {
     /// Default congestion info should be no congestion => maximally permissive.
     #[test]
     fn test_default_congestion() {
-        let inner_congestion_info = CongestionInfoV1::default();
+        let config = CongestionControlConfig::default();
```

Comment:
This test should not use the default config, but rather the one matching PROTOCOL_VERSION.

```suggestion
        let config_store = RuntimeConfigStore::new(None);
        let config = config_store.get_config(PROTOCOL_VERSION);
```